docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+volumes:
+  build:
+
+services:
+
+  server:
+    # Image used for all the other services (proxito, web, celery, build)
+    build:
+      context: ${PWD}
+      dockerfile: ./Dockerfile
+    environment:
+      - FONTAWESOME_NPM_AUTH_TOKEN=${FONTAWESOME_NPM_AUTH_TOKEN}
+    ports:
+      - "8000:8000"
+    volumes:
+      - ${PWD}/:/project
```

Comment:
There is just one container, so we could do two things:

* Maintain YAML setup in docker-compose, keeping in mind the extra dependency (docker-compose is a dependency!)
* Convert to a `docker run` command, store it in a Makefile.

Reply:
I think docker compose is the more expected option, no?

Reply:
Yes, well, you can argue in both directions, but we'll opt for the docker-compose strategy :+1: